calico-cloud/_includes/components/EnvironmentFile.js

```diff
@@ -10,15 +10,16 @@ export default function EnvironmentFile(props) {
   return (
     <>
       <p>
-        <span>
-          Use the following guidelines and sample file to define the environment variables for starting Calico on the
-          host. For more help, see the{' '}
-        </span>
         {props.install === 'container' ? (
-          <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
-        ) : (
-          <Link href={`${baseUrl}/reference/felix/configuration`}>Felix configuration reference</Link>
-        )}
+            <span>
+               Use the following guidelines and sample file to define the environment variables for starting Calico on the
+          host. For more help, see the <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
```

Comment:
The formatting / readability could be improved with aligned indention for the text under the span.

Reply:
Thanks, done!